app/models/course/settings/survey_component.rb

```diff
@@ -9,6 +9,29 @@ def self.email_setting_items
     }
   end
 
+  def lesson_plan_item_settings
```

Comment:
These few methods are repeated similarly across the different settings for survey and video. Can move it out into a concern?

Reply:
If you are going to say that the component_title is different for each setting, then leave that out of the hash in the concern, and do a hash merge in the settings class itself.

Reply:
Done!